Rise-Vision/rise-vision-apps
# Open Time & Date settings

## Description
Opens the ( empty ) Time & Date settings editor and adds the streamline _time_ icon

## Motivation and Context
Time & Date component UI Settings development

## How Has This Been Tested?
Tested locally and on staging

https://apps-stage-8.risevision.com/templates/edit/ba5fb48a-891c-4548-9b49-9e83ad4b885e/?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443

**Note** Labels for the instances will be applied in followup task as this involves applying appropriate label based on `type` of component

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Manually tested
  - Initial automated unit tests added
  - There are no templates visible to users that use _rise-time-date_ component in them, very low risk deployment
  - Rollback will involve re-running last master CCI build followed by reverting code changes in repo
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
- No documentation required
- Support not required
